mdjs-layout/src/mdjs-styles.css.js

```diff
@@ -0,0 +1,117 @@
+export default `
```

Comment:
please use `/* css */`blabllaba`` then you have the highlighting 😎

Reply:
ah nice, done